xivo_dao/alchemy/meeting.py

```diff
@@ -45,6 +46,7 @@ class Meeting(Base):
     guest_endpoint_sip_uuid = Column(UUID(as_uuid=True), ForeignKey('endpoint_sip.uuid', ondelete='SET NULL'))
     tenant_uuid = Column(String(36), ForeignKey('tenant.uuid', ondelete='CASCADE'), nullable=False)
     created_at = Column(DateTime(timezone=True), default=datetime.datetime.utcnow, server_default=text("(now() at time zone 'utc')"))
+    persistent = Column(Boolean, default=False, nullable=False)
```

Comment:
Do you mind if we use `server_default='false'` instead of `default=False`

Small preference to add this logic into database, then if we don't pass through sqlalchemy, we're not required to add this field